### PR TITLE
Use opt-cli

### PR DESCRIPTION
**Pull request by sarbbottam**
Fixed `ghooks` in the [first commit](https://github.com/kentcdodds/eslint-find-new-rules/commit/823e565f9a14190ab613518cea1c0f61feb5cb10).
I am bit confused about the `opt-cli`, please refer the [second commit](https://github.com/kentcdodds/eslint-find-new-rules/commit/a8d3296f078f8fdc9e1c68612cf8a345c1084eba), I have followed the instructions provided at [opt-cli README](https://github.com/ta2edchimp/opt-cli/blob/master/README.md), however could not opt in to `precommit ghooks` even after creating a `.opt-in` file with `precommit`

```
吽 cat .opt-in
precommit
```

Any pointer would be helpful.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,10 +62,15 @@
     ]
   },
   "config": {
-    "commig-msg": "validate-commit-msg",
-    "pre-commit": "npm run validate",
+    "opt": {
```
Comment (kentcdodds) [inline package.json]: Please remove this from the config. This essentially doesn't allow us to have our own preferences.

File: package.json [hunk 2]
```diff
@@ -61,10 +62,15 @@
...
+      "in": [ "precommit" ]
+    },
+    "ghooks": {
+      "commit-msg": "validate-commit-msg",
+      "precommit": "opt --in precommit --exec \"npm run validate\""
```
Comment (kentcdodds) [inline package.json]: The hook is called `pre-commit`

Comment (sarbbottam) [inline package.json]: Opps, need to update the opt-cli README.

**Comment by sarbbottam** [comment]
I guess it would be better to update the CONTRIBUTING.md about the usage.
@kentcdodds

**Comment by kentcdodds** [comment]
Nah, that's probably a README thing :-)

**Comment by sarbbottam** [comment]
Let me update the README

**Comment by kentcdodds** [comment]
Oh, sorry, I thought you were talking about the opt-cli repo. Here this should be a contributing for sure!

**Comment by sarbbottam** [comment]
@kentcdodds @ta2edchimp please have a look. Thanks!

**Comment by kentcdodds** [comment]
LGTM! Someone else should review and merge though.

**Comment by ta2edchimp** [comment]
Tried it. Works. Looks good to me, too. Will merge soon.

**Pull request merged**